dashboard/app/controllers/survey_results_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SurveyResultsController < ApplicationController
+
+  def create
+    # puts params[:survey].to_json
+    @survey_result = SurveyResult.new(survey_result_params)
+    @survey_result.user_id = current_user.id
```

Comment:
yeah we don't handle setting initialization, this works.

btw if a signed in user is required for this you can use `before_filter :authenticate_user!`